pytest.ini

```diff
@@ -47,6 +47,7 @@ markers =
     cluster_management_ops
     data_durability
     data_integrity
+    durability
```

Comment:
so durability and data_durability are diff tests?

Reply:
I will check and will use data_durability if its same

Reply:
using data_durability